.gitignore

```diff
@@ -0,0 +1,13 @@
+*.iml
```

Comment:
A lot of files that should not be uploaded to the source control are uploaded, this means the git ignore file is not configured properly. please use the one in the guidelines